crates/rpc/rpc-types/src/eth/engine/payload.rs

```diff
@@ -413,10 +418,15 @@ pub enum PayloadValidationError {
         /// The state root of the payload that we computed locally.
         local: H256,
     },
+    /// Thrown when some other error occured while processing payload
+    #[error("Other")]
```

Comment:
this should delegate the error message 
```suggestion
    #[error("{0}")]
```